ladybug/src/main/resources/springIbisTestTool.xml

```diff
@@ -79,7 +79,7 @@
 
 	<bean name="testStorage,testStoragexxx,testStorageLOC" class="nl.nn.testtool.storage.xml.XmlStorage" autowire="byName" init-method="init">
 		<property name="name" value="Test"/>
-		<property name="metadataFile" value="${ibistesttool.directory}/metadata.xml"/>
+		<property name="metadataFile" value="${log.dir}/testtool4${instance.name.lc}.metadata.xml"/>
 		<property name="reportsFolder" value="${ibistesttool.directory}"/>
```

Comment:
Wat is de overweging om de één in log.dir te hebben, en de ander in ibistesttool.directory?

Reply:
In de reportsFolder zit per report die in de Test tab zit (dus niet de Debug tab) een report xml, deze files wil je als Frank! ontwikkelaar in Git opslaan (net zoals je bij Larva de scenario files). De metadata xml bevat metadata van al deze bestanden en is volledig opgebouwd uit de data die uit deze bestanden komt, feitelijk is het een soort cache bestand zodat niet bij iedere actie alle (of een aantal) report xml's gelezen hoeven te worden. Het is niet nodig (en ook niet handig) om hem in Git op te slaan. Als hij niet betaat wordt hij automatisch aangemaakt en gevuld. Vandaar dat ik hem nu verplaats naar de log.dir waar ook de bestanden voor de Debug tab staan die niet in Git komen.